robotpy_installer/cli_init.py

```diff
@@ -30,7 +30,7 @@ def run(self, main_file: pathlib.Path, project_path: pathlib.Path):
 
             logger.info("Created empty %s", main_file)
 
-        # Create pyproject.toml if it doesn't already exist
+        # Create pyproject.toml and .gitignore if it doesn't already exist
```

Comment:
As written, this will always write a .gitignore (overwriting one if it does exist), iff a pyproject.toml doesn't exist. This doesn't match the comment, and I think the comment is closer to the intent.